incubator/binding-kafka-grpc/src/main/java/io/aklivity/zilla/runtime/binding/kafka/grpc/internal/stream/KafkaGrpcRemoteServerFactory.java

```diff
@@ -1138,7 +1114,7 @@ private void doKafkaWindow(
             }
         }
 
-        private void doKafkaTombstone(
+        private void doDataNull(
```

Comment:
Suggest rename to `doKafkaDataNull` in keeping with `doKafkaBegin` and `onKafkaReset`, etc.